xml/System.Data.Objects.DataClasses/EntityCollection`1.xml

```diff
@@ -463,7 +463,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This method calls the <xref:System.Data.Objects.DataClasses.RelatedEnd.ValidateLoad%2A> method before loading the collection. See this method for a list of possible exceptions.  
+ This method calls the <xref:System.Data.Objects.DataClasses.RelatedEnd.Load*> method before loading the collection. See this method for a list of possible exceptions.  
```

Comment:
Maybe I'm missing something but I don't see Load calling RelatedEnd.Load. It's actually calling an internal Load method on the class itself. Given that this page exists as preview only, it's possible that the actual implementation changed from when the docs were written and the product shipped. Perhaps we should remove this paragraph?